main_cron.py

```diff
@@ -46,6 +46,12 @@
         cron.CronActivitySearchRankHandler),
     main.get_redirect_route(
         r'/cron/jobs/cleanup', cron.CronMapreduceCleanupHandler),
+    main.get_redirect_route(
+        r'/cron/mail/admins/contributor_dashboard_review_issues',
```

Comment:
Bit confused about the naming here. What are "review issues"?

Maybe /contributor_dashboard_issues_to_review?

(Ditto for the handler name in the line below.)

Reply:
I was thinking review turnaround time issues..I think I'll put that since it's more specific?

Reply:
I don't know what "review turnaround time issues" means either. "Issues" is also a term of art used in the lesson analytics tab.

Given what this handler appears to be doing, perhaps you could call it "contributor_dashboard_bottlenecks" to avoid using verbs in the name.

Reply:
updated